tools/server-spring/src/main/java/org/eclipse/rdf4j/common/webapp/system/logging/LoggingOverviewController.java

```diff
@@ -41,9 +41,9 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
 		Map<String, Object> model = new HashMap<>();
 		LogReader logReader = getLogReader(offset, count, request);
 		model.put("logreader", logReader);
-		model.put("offset", new Integer(offset));
```

Comment:
Just nitpicking here: this change is not about UTF8 encoding. But OK for me.

Reply:
Ah yeah, got a bit carried away with cleanup.